action.yml

```diff
@@ -29,50 +29,63 @@ inputs:
 runs:
   using: "composite"
   steps:
-  - name: Setup GITHUB_PATH for script
-    run: |
-      echo "${{ github.action_path }}" >> $GITHUB_PATH
-      echo "${{ inputs.working-directory }}"
-    shell: bash
-  - name: Grant permissions
-    run: chmod +x ${GITHUB_ACTION_PATH}/install.sh
-    shell: sh
-  - id: keploy-test-report
-    name: Run Script
-    run:  |
-      ${GITHUB_ACTION_PATH}/install.sh
-      ${GITHUB_ACTION_PATH}/install.sh > ${GITHUB_WORKSPACE}/${WORKDIR}/report.txt
-      cat ${GITHUB_WORKSPACE}/${WORKDIR}/report.txt
+    - name: Setup GITHUB_PATH for script
+      run: |
+        echo "${{ github.action_path }}" >> $GITHUB_PATH
+        echo "${{ inputs.working-directory }}"
+      shell: bash
+    - name: Grant permissions
+      run: chmod +x ${GITHUB_ACTION_PATH}/install.sh
+      shell: sh
+    - id: keploy-test-report
```

Comment:
there is an extra indentation due to which the workflow is failing !

Reply:
Resolved

Reply:
They are still there

Reply:
According to my testing, the reason the workflow was failing as the delay was set to ```10``` instead of ```10s```. The indentation is correct according to online yaml checks. Could you kindly test the workflow again?